speechbrain/run_conformer.sh

```diff
@@ -2,27 +2,32 @@
 
 export PYTHONPATH="..":$PYTHONPATH
 
-SOURCE="speechbrain/asr-conformer-transformerlm-librispeech"
+SOURCE="speechbrain/asr-conformer-largescaleasr"
 
+# Run with CTC+Attn
 python run_eval.py \
   --source=$SOURCE \
   --speechbrain_pretrained_class_name="EncoderDecoderASR" \
   --dataset_path="hf-audio/esb-datasets-test-only-sorted" \
   --dataset="librispeech" \
   --split="test.clean" \
-  --device=0 \
+  --device=7 \
   --batch_size=4 \
-  --max_eval_samples=-1
+  --max_eval_samples=-1 \
+  --beam_size=40 \
```

Comment:
Isn;t this too high of a beam size :D (can we do with 5, curious as 40 would take lot of time to eval)